html/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="./favicon.svg" type="image/svg+xml">
```

Comment:
You must be joking!!!!!

index.html how???? Go and fix this, you're writing a next project

Reply:
this is not coming from the component i worked on but the one i pulled from upstream